### ISSUE TITLE
Early stopping handler should fire on Events.EPOCH_COMPLETED

### ISSUE THREAD

**Issue by pbamotra** [pytorch/ignite#271]
Hi guys,

I was going through the handlers section and I feel that early stopping should be called on Events.EPOCH_COMPLETED to check if validation loss / accuracy hasn't improved unlike documentation where it says to fire on Events.COMPLETED?

Am I right?

**Comment by pbamotra** [comment]
nvm, it's already asked in closed issues.

**Comment by vfdev-5** [comment]
Thanks for the feedback @pbamotra. Yes it is duplicate of #258.

However we need to update the docs
